API/Models/AccountModels/LoginModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace API.Models.AccountModels
+{
+    public class LoginModel
+    {
+        [Required]
+        [EmailAddress]
+        public string Email { get; set; }
+
+        [Required]
+        public string Password { get; set; }
+
+        public bool RememberMe { get; set; }
```

Comment:
This is not needed

Reply:
It is currently used in the Login request. The idea is that we could create different account models for each method of login. For example, this one requires Email, but we could create another model such as LoginWithUsernameModel. I think it would make the code somewhat cleaner than having methods with just string parameters.